packages/exp/src/expo_commands/generate-unimodule.js

```diff
@@ -0,0 +1,148 @@
+/**
+ * @flow
+ */
+import path from 'path';
+import proc from 'child_process';
+import prompt from '../prompt';
+import targz from 'targz';
+import fs from 'fs';
+import copy from 'recursive-copy';
+import fsExtra from 'fs-extra';
+import replace from 'replace';
+import os from 'os';
+
+const NPM_TEMPLATE_VERSION = '1.0.1';
+const TEMP_DIR_NAME = `temp-expo-module-template`;
+
+const decompress = async() => {
+  return new Promise((resolve, reject) => {
+    targz.decompress({
+        src: `expo-module-template-${NPM_TEMPLATE_VERSION}.tgz`,
+        dest: TEMP_DIR_NAME,
+      },
+      error => {
+        if (error) {
+          reject(error);
+        } else {
+          resolve(null);
+        }
+      }
+    );
+  });
+};
+
+export default (program: any) => {
+  program
+    .command('generate-unimodule')
+    .description('Generate new unimodule.')
+    .asyncAction(async () => {
+      const configuration = {
+        jsName: null,
+        podName: null,
+        javaModule: null,
+      };
+
+      configuration.jsName = (await prompt({
+        message: 'How would you like to call your module in JS/NPM? (eg. expo-camera)',
+        name: 'jsName'
+      })).jsName;
+
+      configuration.podName = (await prompt({
+        message: 'How would you like to call your module in Cocoapods? (eg. EXCamera) (leave empty to not include iOS part)',
+        name: 'podName'
+      })).podName;
+
+      configuration.javaModule = (await prompt({
+        message: 'How would you like to call your module in Java? (eg. expo.modules.camera)',
+        name: 'javaModule'
+      })).javaModule;
+
+      proc.execSync(`npm pack expo-module-template@${NPM_TEMPLATE_VERSION}`);
```

Comment:
Shouldn't we use the latest version? Otherwise updating the template will require us to release a new version of Expo CLI and users to update to it.

Reply:
I added [ ^ ] before version so that we could fix the bugs in template without releasing a new version of Expo CLI.

Reply:
👍